packages/vats/decentral-demo-config.json

```diff
@@ -49,9 +49,6 @@
     "sharing": {
       "sourceSpec": "@agoric/vats/src/vat-sharing.js"
     },
-    "walletManager": {
-      "sourceSpec": "@agoric/vats/src/vat-walletManager.js"
-    },
```

Comment:
this config probably needs the same `smart-wallet.js` contract bundle as `decntral-core-config.json`.